tests/helpers/perf_run_helper.rb

```diff
@@ -8,7 +8,7 @@ module PerfRunHelper
   BASELINE_PE_VER = ENV['BASELINE_PE_VER']
   SCALE_ITERATIONS = 10
   SCALE_INCREMENT = 100
-  PUPPET_METRICS_COLLECTOR_SERVICES = %w[orchestrator puppetdb puppetserver]
+  PUPPET_METRICS_COLLECTOR_SERVICES = ENV['BEAKER_INSTALL_TYPE'] == 'pe' ? %w[orchestrator puppetdb puppetserver] : ['puppetserver']
```

Comment:
I know that at least the orchestrator isn't installed in foss, and I'm currently only concerned about server.

Same comment about a more reliable `get_puppet_server_service_name_from_env`